app/src/main/java/org/mozilla/fenix/settings/HomeSettingsFragment.kt

```diff
@@ -62,6 +63,15 @@ class HomeSettingsFragment : PreferenceFragmentCompat() {
             onPreferenceChangeListener = CustomizeHomeMetricsUpdater()
         }
 
+        requirePreference<Preference>(R.string.pref_key_wallpapers).apply {
+            setOnPreferenceClickListener {
+                view?.findNavController()?.navigate(
+                    HomeSettingsFragmentDirections.actionHomeSettingsFragmentToWallpaperSettingsFragment()
+                )
+                true
+            }
+        }.isVisible = FeatureFlags.showWallpapers
```

Comment:
Couldn't the `isVisible` be inside the `apply { }`?

Reply:
Oh yeah, it definitely should be. Updated